src/env/env.rs

```diff
@@ -168,6 +168,8 @@ pub fn uumain(args: Vec<String>) -> i32 {
         item = iter.next();
     }
 
+    let win_comspec = if !cfg!(windows) { "".to_string() } else { env::var("ComSpec").unwrap_or("cmd".to_string()) };
```

Comment:
I'd prefer if the order was reversed (_i.e._ make the `else` clause `"".to_string()`).  This could also probably just configured to only compile on Windows (to avoid the empty string allocation on Unix).

Reply:
I refactored it.